oft-live-demo-medium.md

```diff
@@ -222,47 +279,55 @@ Sounds familiar?
 Let's take this one step further and filter a single result including details from the report.
 
 ```bash
-id='req~tracing.deep-coverage~1' oft trace -v ALL doc src/main/java src/test/java | grep -Pzo 'ok.*'"$id"'.*\n([|].*\n)+'
+id='req~tracing.deep-coverage~1'
+oft trace -v all doc/spec $(find . -name java) | grep -Pzo 'ok.*'"$id"'.*(\n| .*\n)+'
 ```
+
+This yields:
+
+    ok [ in:  1 /  1 ✔ | out:  1 /  1 ✔ ] req~tracing.deep-coverage~1 (dsn)
+    
+    OFT marks a specification item as _covered deeply_ if this item - and all items it needs coverage from - are covered recursively.
+    
+    [covered shallow  ] ← dsn~tracing.deep-coverage~1
+    [covers           ] → feat~requirement-tracing~1
+
 If you think now: this command is madness - fair enough. The point is that through chaining OFT with other tools you can do things the designers of OFT didn't think of.
 
 ### Converting Specification Formats
+
 While the importer of OFT automatically detects the format you feed into a trace, you might need to deliver specifications in a certain format to another organization.
 
 OFT can work as converter. In this case the original content is converted into the target format of your choice.
 
 The following command converts the system requirements and design of OFT into the ReqM2 format.
 
 ```bash
-oft convert -o specobject doc
+oft convert -o specobject doc/spec
 ```
 
-*(The next step assumes you have `xmllint` installed. Of course you can use other software to pretty print the XML output too)*
+*(The next step assumes you have `xmllint` installed. Of course, you can use other software to pretty print the XML output too)*
 
 ```bash
-oft convert -o specobject doc | xmllint --format - > /tmp/oft_specifications.xml
+oft convert -o specobject doc/spec | xmllint --format - > /tmp/oft_specifications.xml
 ```
 
 ```bash
 vi /tmp/oft_specifications.xml
 ```
 
 ### Which Exchange Formats Does OFT support?
+
 * OFT native: Requirement-enhanced Markdown (in / out)
 * ReqM2 (in / out)
 * OFT code decoration (in)
 * Reqm2 code decoration (in)
 
-Planned:
-
-* RIFF (in)
-* CSV (out)
-
 ### Where to go From Here?
 You can find documentation about OpenFastTrace on GitHub:
 
 * [OpenFastTrace project page](https://github.com/itsallcode/openfasttrace)
 * [Contributing to OFT](https://github.com/itsallcode/openfasttrace/CONTRIBUTING.md)
 
 Handbook is work in progress (at the time of writing this script 2018-05-20):
-* [User guide](https://github.com/itsallcode/openfasttrace/blob/story/%23111-Add_handbook/doc/user_guide.md)
+* [User guide](https://github.com/itsallcode/openfasttrace/blob/story/%23111-Add_handbook/doc/user_guide.md)
```

Comment:
User guide is already finished: https://github.com/itsallcode/openfasttrace/blob/main/doc/user_guide.md